packages/grafana-data/src/utils/url.ts

```diff
@@ -199,19 +199,12 @@ export const urlUtil = {
  * Create an string that is used in URL to represent the Explore state. This is basically just a stringified json
  * that is that used as a state of a single Explore pane so it does not represent full Explore URL.
  *
- * There are 2 versions of this, normal and compact. Normal is just the same object stringified while compact turns
- * properties of the object into array where the order is significant.
  * @param urlState
- * @param compact
+ * @param compact this parameter is deprecated and will be removed in a future release.
```

Comment:
What do we need to do before we remove compact mode? Migrate short links?

Reply:
nothing actually, the parsing logic will still work with compact URLs. when we then decide to completely remove support for compact URLs we only need to make sure explore doesn't break.